app/configs.py

```diff
@@ -86,79 +42,87 @@ class _BaseConfig:
     PROXY_INFO_FILE: str = "/boot/ota/proxy_info.yaml"
     PASSWD_FILE: str = "/etc/passwd"
     GROUP_FILE: str = "/etc/group"
-    BOOT_OTA_PARTITION_FILE: str = "ota-partition"
+
+    # status files
     OTA_STATUS_FNAME: str = "status"
     OTA_VERSION_FNAME: str = "version"
+    SLOT_IN_USE_FNAME: str = "slot_in_use"
+
     LOG_FORMAT: str = (
         "[%(asctime)s][%(levelname)s]-%(filename)s:%(funcName)s:%(lineno)d,%(message)s"
     )
+
+    # standby/refroot mount points
     MOUNT_POINT: str = "/mnt/standby"
+    # where active(old) image partition will be bind mounted to
+    REF_ROOT_MOUNT_POINT: str = "/mnt/refroot"
 
     # ota-client behavior setting
     CHUNK_SIZE: int = 1 * 1024 * 1024  # 1MB
+    LOCAL_CHUNK_SIZE: int = 4 * 1024 * 1024 # 4MB
     DOWNLOAD_RETRY: int = 5
     DOWNLOAD_BACKOFF_MAX: int = 10 # seconds
     MAX_CONCURRENT_DOWNLOAD: int = 8
     MAX_CONCURRENT_TASKS: int = 1024
     STATS_COLLECT_INTERVAL: int = 1 # second
+    ## standby creation mode
+    STANDBY_CREATION_MODE: Literal["legacy", "rebuild", "in-place", "auto"] = "auto"
```

Comment:
What is auto?
```
@enum
calss StandbyCreationModes:
    LEGACY = auto()
    REBULD = auto()
    IN_PLACE = auto()
```
and then
```
STANDBY_CREATION_MODE = StandbyCreationModes.REBUILD
```
is clear.

Reply:
Ah that is a not yet introduced feature that choose create standby mechanism automatically.
Since I am not going to implement the auto selection mechanism now, I will do as you said.

Reply:
I'm not sure what is the `auto` but the static mode is fine, I think.

Reply:
updated in f76efed